src/Game.h

```diff
@@ -179,22 +178,6 @@ class Game
 	[[nodiscard]] Camera& GetCamera() const { return *_camera; }
 	[[nodiscard]] Sky& GetSky() const { return *_sky; }
 	[[nodiscard]] Water& GetWater() const { return *_water; }
-	LandIsland& GetLandIsland()
-	{
-		if (_landIsland == nullptr)
-		{
-			throw std::runtime_error("Cannot get landscape before any are loaded");
-		}
-		return *_landIsland;
-	}
-	[[nodiscard]] LandIsland& GetLandIsland() const
-	{
-		if (_landIsland == nullptr)
-		{
-			throw std::runtime_error("Cannot get landscape before any are loaded");
-		}
```

Comment:
This error used to be thrown

Reply:
In regards to this i can only thing of adding a check somewhere to make sure that the landIsland is properly loaded, but i don't know any other way than checking every place the Game::Instance()->GetLandIsland() was called and checking if the Locator::terrainSystem::hasValue() and throwing the exception.
Do you have any other idea of what could be done?

Reply:
I don't think every call to GetLandIsland needs to be checked but there are certain places we know that it can happen